code/Solidity/Faucet.sol

```diff
@@ -1,9 +1,12 @@
 // Version of Solidity compiler this program was written for
-pragma solidity 0.5.12;
+pragma solidity ^0.6;
 
 // Our first contract is a faucet!
 contract Faucet {
 
```

Comment:
I think, can remove the blank line.
Because there is no such an extra line break in https://solidity.readthedocs.io/en/v0.6.2/solidity-by-example.html#voting
and https://solidity.readthedocs.io/en/v0.6.2/style-guide.html

Reply:
Hey chris, that sounds great. go ahead and make those changes